pkg/didcomm/packager/package_test.go

```diff
@@ -289,6 +289,6 @@ func (m *mockProvider) InboundTransportEndpoint() string {
 	return "sample-endpoint.com"
 }
 
-func (m *mockProvider) Packer() packer.Packer {
-	return m.outboundPacker
+func (m *mockProvider) PrimaryPacker() packer.Packer {
+	return m.packer
```

Comment:
might as well call the member field `primaryPacker`

Reply:
Done

Reply:
done